chemprop/args.py

```diff
@@ -84,6 +84,11 @@ class CommonArgs(Tap):
     """Number of workers for the parallel data loading (0 means sequential)."""
     batch_size: int = 50
     """Batch size."""
+    atom_descriptors: Literal['None', 'feature', 'descriptor'] = 'None'
+    """Custom extra atom descriptors. feature: used as atom features to featurize a given molecule. 
+    descriptor: used as descriptor and concatenated to the machine learned atomic representation"""
```

Comment:
I'm just changing the formatting a little bit for consistency and so that the docstring shows up nicely on the documentation website (https://chemprop.readthedocs.io/en/latest/args.html).

```suggestion
    """
    Custom extra atom descriptors.
    :code:`feature`: used as atom features to featurize a given molecule. 
    :code:`descriptor`: used as descriptor and concatenated to the machine learned atomic representation.
    """
```